### ISSUE TITLE
Example for collections?

### ISSUE THREAD

**Issue by miguelslp** [d3v1an7/testing-11ty-webc-for#1]
Nice work here, it's a bit hard to find webc:for examples out there. Could you please show one accessing 11ty collections? 
I saw an example here but couldn't get it to work
https://github.com/darthmall/11ty.webc.fun/blob/main/src/_includes/components/card-carousel.webc

**Comment by d3v1an7** [comment]
Hey @miguelslp! The example you linked to was how loops were being handled prior to the introduction of `webc:for`, which is really new.

The example in my repo should work as-is, the intent of the repo is to [highlight the issue I raised here](https://github.com/11ty/webc/issues/28#issuecomment-1482019076), which now lives as a [standalone issue here](https://github.com/11ty/eleventy-plugin-webc/issues/67).

In short, everything _should_ work [as per docs here](https://www.11ty.dev/docs/languages/webc/#webcfor-loops), but you have to reset the server each time to see any file changes within components.

What specific issues are you running in to?

**Comment by miguelslp** [comment]
You are right, it should work as-is, I was facing some issues but got it to work in the end, thanks anyway!
